### ISSUE TITLE
您好，出现下面这个警告该如何修改代码里面的内容呢？torch文档说换某一步骤的内容，但不太清楚如何更换。

### ISSUE THREAD

**Issue by liqianqi** [dog-qiuqiu/Yolo-FastestV2#51]
UserWarning: Detected call of `lr_scheduler.step(()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  a
Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/opt.im.html#how-to-adjust-learning-rate
